### PR TITLE
a11y:notification

### PR DESCRIPTION
**Pull request by DaiQiangReal**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [ ] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes #

### Changelog
🇨🇳 Chinese
- 修复 ...

---

🇺🇸 English
- Fix ...

### Checklist
- [ ] Test or no need
- [ ] Document or no need
- [ ] Changelog or no need

### CODE REVIEW

File: packages/semi-ui/notification/notice.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ import BaseComponent from '../_base/baseComponent';
 import { isSemiIcon } from '../_utils';
 import { noop } from 'lodash';
 import { IconAlertTriangle, IconInfoCircle, IconTickCircle, IconAlertCircle, IconClose } from '@douyinfe/semi-icons';
+import { getUuidv4 } from '@douyinfe/semi-foundation/utils/uuid';
```
Comment (shijiatongxue) [inline packages/semi-ui/notification/notice.tsx:17]: 使用 getUuidShort 吧，这个更短

Comment (shijiatongxue) [reply packages/semi-ui/notification/notice.tsx:17]: 在 feat/a11y-aria 分支

File: packages/semi-ui/notification/notice.tsx [hunk 2]
```diff
@@ -152,19 +153,21 @@ class Notice extends BaseComponent<NoticeReactProps, NoticeState> {
             [`${prefixCls}-${theme}`]: theme === 'light',
             [`${prefixCls}-rtl`]: direction === 'rtl',
         });
+        const titleID = getUuidv4();
         return (
             <div
```
Comment (shijiatongxue) [inline packages/semi-ui/notification/notice.tsx:158]: Notification 加个 role=alert 是不是合适？

File: packages/semi-ui/notification/notice.tsx [hunk 3]
```diff
@@ -152,19 +153,21 @@ class Notice extends BaseComponent<NoticeReactProps, NoticeState> {
...
                 onClick={this.notifyClick}
+                aria-labelledby={titleID}
             >
                 <div>{this.renderTypeIcon()}</div>
                 <div className={`${prefixCls}-inner`}>
-                    <div className={`${prefixCls}-content-wrapper`}>
-                        {title ? <div className={`${prefixCls}-title`}>{title}</div> : ''}
-                        {content ? <div className={`${prefixCls}-content`}>{content}</div> : ''}
+                    <div className={`${prefixCls}-content-wrapper`} aria-label={'wrapper'}>
+                        {title ? <div id={titleID} className={`${prefixCls}-title`} aria-label={'Notification title'}>{title}</div> : ''}
```
Comment (shijiatongxue) [inline packages/semi-ui/notification/notice.tsx:169]: 这里的 aria-label 感觉不是很有必要

File: packages/semi-ui/notification/notice.tsx [hunk 4]
```diff
@@ -152,19 +153,21 @@ class Notice extends BaseComponent<NoticeReactProps, NoticeState> {
...
+                        {content ? <div className={`${prefixCls}-content`} aria-label={'Notification content'}>{content}</div> : ''}
```
Comment (shijiatongxue) [inline packages/semi-ui/notification/notice.tsx:170]: +1，这里的 content 感觉不需要加 aria-label

File: packages/semi-ui/notification/notice.tsx (hunk 3 above)
Comment (DaiQiangReal) [reply packages/semi-ui/notification/notice.tsx:169]: title 还是 wrapper ？

**Pull request merged**